kubernetes/README.md

```diff
@@ -18,32 +18,34 @@ But do not merge yet.
-    * If there are any manifest updates apart from the version update from the main [CoreDNS deployment](https://github.com/coredns/deployment/blob/master/kubernetes/coredns.yaml.sed), include them in the PR. 
+5. Once the gcr.io image has been pushed, open a PR in the [Kubernetes main repository](https://github.com/kubernetes/kubernetes) to update the CoreDNS image.
+for both [kube-up](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns/coredns) and [kubeadm](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/constants/constants.go#L335) install tools.
+    * Make sure to write proper release notes mentioning the new version, features, deprecation as applicable from the previous release.
     * Ensure that all the pre-submit e2e tests are passing. 
     * If the tests are passing, get the PR merged.
+    
+6. If there are any manifest updates apart from the version update from the main [CoreDNS deployment](https://github.com/coredns/deployment/blob/master/kubernetes/coredns.yaml.sed), include them in the PR. 
+    * Sync with the kube-dns manifests to ensure that there are no important updates missing.
+    * If the manifest updates contain any user-facing changes, write clear release-notes in the PR stating why the changes are being made.
 
 
 6. Validate the [5k node scale test](https://k8s-testgrid.appspot.com/sig-scalability-gce#gce-scale-performance) to see if the Load and Density scale tests are green. 
     * The 5k node scale test only runs once per day. Hence ensure that the validated test run includes the merged PR. 
-    * If the tests are green, proceed to open a PR in the [Minikube](https://github.com/kubernetes/minikube) and [Kops](https://github.com/kubernetes/kops) repository to update the CoreDNS image and manifest.
+    * If the tests are green, proceed to open a PR in the [Kops](https://github.com/kubernetes/kops) repository to update the CoreDNS image and manifest.
 
 
 
 ## Checklist (must be completed in order)
 
-- [ ] Open a coredns/deployment PR
+- [ ] Open a coredns/deployment PR to update the CoreDNS image
 - [ ] coredns/deployment passes CI tests
 - [ ] Open a temporary kubernetes/kubernetes PR to test the (coredns/coredns:version) image
 - [ ] kubernetes/kubernetes passes e2e dns pre-submit conformance tests
 - [ ] Close the temporary kubernetes/kubernetes PR
 - [ ] Merge the coredns/deployment PR
 - [ ] Open a kubernetes/kubernetes issue to push CoreDNS image to k8s.gcr.io
-- [ ] Open a kubernetes/kubernetes PR to update manifests and the (k8s.gcr.io/coredns:version) image
-- [ ] kubernetes/kubernetes update PR merged
+- [ ] Open a kubernetes/kubernetes PR to update the (k8s.gcr.io/coredns:version) image
+- [ ] kubernetes/kubernetes image update PR merged
 - [ ] Verify that gce-scale-performance test passes (runs once daily)
-- [ ] Open PRs for minikube & kops
-
-
-
+- [ ] Open a kubernetes/kubernetes PR to update the manifests
```

Comment:
why are we doing this as a separate step?

Reply:
We can do it in the same PR, but I think separating the PRs is a good idea so that the release notes for the version difference and adding updates to the manifest can be kept separate.

Reply:
What is the benefit of separating the release notes?

Reply:
I'll revert it to how it was before.